pkg/apis/kubeone/helpers.go

```diff
@@ -51,6 +51,10 @@ func (c KubeOneCluster) Followers() []HostConfig {
 	return followers
 }
 
+func (c KubeOneCluster) StaticWorkers() []HostConfig {
+	return c.WorkerHosts
```

Comment:
how about, rename `WorkerHosts` to `StaticWorkers`, and then we wouldn't need trivial getter